### PR TITLE
Admin view exams

### PR DESCRIPTION
**Pull request by tranduytung**

### CODE REVIEW

**Comment by kagoromo** [comment]
OK

**Comment by qmau94** [comment]
ok

File: app/controllers/admins/exams_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Admins::ExamsController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @subjects = Subject.all
+    @status = Exam.statuses
```
Comment (hellovietnam93) [inline app/controllers/admins/exams_controller.rb]: số ít à em?

File: app/controllers/admins/exams_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    @search = @exams.ransack params[:q]
+    if params[:q].nil?
```
Comment (hellovietnam93) [inline app/controllers/admins/exams_controller.rb]: @exams = if .. else ..end

**Pull request merged**